app/src/main/res/layout-sw600dp-land/topic_revision_summary_view.xml

```diff
@@ -1,67 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<layout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto">
-
-  <data>
-
-    <import type="android.view.View" />
-
-    <import type="org.oppia.android.app.model.Subtopic" />
-
-    <variable
-      name="viewModel"
-      type="org.oppia.android.app.topic.revision.revisionitemviewmodel.TopicRevisionItemViewModel" />
-  </data>
-
-  <com.google.android.material.card.MaterialCardView
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:layout_marginStart="32dp"
-    android:layout_marginTop="24dp"
-    android:layout_marginEnd="32dp"
-    android:clipToPadding="true"
-    android:onClick="@{(v) -> viewModel.onRevisionItemPressed.onTopicRevisionSummaryClicked(viewModel.subtopic)}"
-    app:cardBackgroundColor="@color/white"
-    app:cardCornerRadius="4dp"
-    app:cardElevation="4dp"
-    app:cardMaxElevation="4dp">
-
-    <androidx.constraintlayout.widget.ConstraintLayout
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content">
-
-      <org.oppia.android.app.customview.LessonThumbnailImageView
-        android:id="@+id/subtopic_image_view"
-        android:layout_width="0dp"
-        android:layout_height="0dp"
-        android:importantForAccessibility="no"
-        android:scaleType="centerInside"
-        app:entityId="@{viewModel.topicId}"
-        app:entityType="@{viewModel.entityType}"
```

Comment:
This information about the image scaleType is not taken care of while merging. Also I would like to clarify one thing with @rt4914 that why only `land` version of xml has scaleType attribute but not the `portrait` one.

Reply:
You can either keep two files separate for sw600dp & mobile layout and then include scaleType only in the sw600dp ones OR you can include scaleType here normally that will be applied to all the versions of screen. The latter seems good to me but it can possibly include a change in UI, we will have to confirm this with @rt4914 before proceeding.